src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -42,7 +42,20 @@ object WorkflowActor {
 
   val DatabaseTimeout = 5 seconds
 
-  type ExecutionStore = Map[Call, ExecutionStatus.Value]
+  type ExecutionStore = Map[ExecutionStoreKey, ExecutionStatus]
+  type ExecutionStoreEntry = (ExecutionStoreKey, ExecutionStatus)
```

Comment:
I'm liking this `ExecutionStore` with the possible exception of my `parent` nit above.  I de-`Map`'d the store locally and found myself soon regretting it.